backend/compound_library/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "compound_library"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+sea-orm ={ workspace = true, features = ["sqlx-postgres"]}
+axum = { workspace = true }
+tokio ={ workspace = true }
+sea-orm-migration ={ workspace = true }
+async-graphql ={ workspace = true }
+graphql_endpoints = { path = "../graphql_endpoints" }
+dotenvy ={ workspace = true }
+clap ={ workspace = true }
+opa_client ={ path = "../opa_client", features = ["graphql"]}
+url ={ workspace = true }
+tracing = { workspace = true }
+tracing-subscriber = { workspace = true }
+uuid = { workspace = true , features = ["v7"]}
+the_paginator = { version = "0.1.0", path = "../the_paginator", features = [
+    "async-graphql",
+] }
```

Comment:
Formatting is a bit weird here, any chance you could sort it alphabetically and format it with Tapio / VSCode Even better TOML (`tamasfe.even-better-toml`)